diff --git a/meson.build b/meson.build
index 96a0a8c..964da4d 100644
--- a/meson.build
+++ b/meson.build
@@ -88,18 +88,6 @@ if historydir == ''
     historydir = get_option('prefix') / get_option('localstatedir') / 'lib' / 'upower'
 endif
 
-udevrulesdir = get_option('udevrulesdir')
-if udevrulesdir == 'auto'
-    udev_dep = dependency('udev', required: true)
-    udevrulesdir = udev_dep.get_variable(pkgconfig: 'udev_dir') / 'rules.d'
-endif
-
-udevhwdbdir = get_option('udevhwdbdir')
-if udevhwdbdir == 'auto'
-    udev_dep = dependency('udev', required: true)
-    udevhwdbdir = udev_dep.get_variable(pkgconfig: 'udev_dir') / 'hwdb.d'
-endif
-
 dbusdir = get_option('datadir') / 'dbus-1'
 systemdsystemunitdir = get_option('systemdsystemunitdir')
 if systemdsystemunitdir == ''
